chains/xcm-integration-tests/emulated/chains/orchestrator-paras/dancebox-rococo/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
```

Comment:
I would say this one is non-needed and we can work only with "dancebox". I dont see substantial differences between dancebox and danceboxrococo..

Reply:
Nuked it. The less code - the better :)